portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +41,20 @@ function writeText(inputText, container) {
     typeWriter();
 }
 
-/** Function that calls writeText for the welcome messages. */
+/** Function that calls writeText for the welcome messages. Ensures that no greeting is repeated twice in a row.*/
+var oldIndex = 0;
```

Comment:
might want to highlight this change in your PR description

Reply:
Oh yeah, I completely forgot that I did this yesterday! Just updated!